### ISSUE TITLE
Fix WebMock / HTTP gem warning

### ISSUE THREAD

**Issue by mguterl** [gaslight/gaslight#146]
`You are using HTTP Gem 0.5.0. WebMock supports version >= 0.6.0.`

**Comment by mguterl** [comment]
The blog_app engine depends on the engagement gem, which depends on the twitter gem. engagement does not specify which version of twitter to use. I tried to run `bundle update twitter` inside of the blog_app engine, but it started updated Rails dependencies which made me nervous.

**Comment by st23am** [comment]
I got as far updating the dependencies of the engagement gem and then got blocked on the hacker_news_search gem depending on an older version of faraday.

Steps taken

```
 ~/src/gaslight/gaslight (master*) $ bundle update http
Fetching gem metadata from https://rubygems.org/.........
Fetching additional metadata from https://rubygems.org/..
Resolving dependencies...
Bundler could not find compatible versions for gem "http":
  In Gemfile:
    http (~> 0.6.0) ruby

    blog_app (>= 0) ruby depends on
      engagement (>= 0) ruby depends on
        twitter (>= 0) ruby depends on
          http (0.5.0)
```

I edited `engagement.gemspec` to include `twitter ~> 0.5.9` which is the first versions to require 0.6.0 of the http gem.

````
# coding: utf-8
lib = File.expand_path('../lib', __FILE__)
$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
require 'engagement/version'

Gem::Specification.new do |spec|
  spec.name          = "engagement"
  spec.version       = Engagement::VERSION
  spec.authors       = ["Michael Guterl"]
  spec.email         = ["michael@diminishing.org"]
  spec.description   = %q{A ruby gem for measuring the engagement of URLs on various social sites. It wont find the love of your life, but it will show you who's talking about you!}
  spec.summary       = %q{Get comment counts from Reddit, HackerNews, Disqus and Twitter.}
  spec.homepage      = "https://github.com/gaslight/engagement"
  spec.license       = "MIT"

  spec.files         = `git ls-files`.split($/)
  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
  spec.require_paths = ["lib"]

  spec.add_development_dependency 'bundler', '~> 1.3'
  spec.add_development_dependency 'rake'
  spec.add_development_dependency 'rspec'
  spec.add_development_dependency 'vcr'
  spec.add_development_dependency 'webmock'
  spec.add_development_dependency 'debugger'
  spec.add_dependency 'hacker_news_search', '~> 0.0.7'
  spec.add_dependency 'snoo'
  spec.add_dependency 'disqus'
  spec.add_dependency 'twitter', '~> 5.9.0'
end
```
Doing a bundle install on the engagement gem gave me:

```
Resolving dependencies...
Bundler could not find compatible versions for gem "faraday":
  In Gemfile:
    engagement (>= 0) ruby depends on
      hacker_news_search (~> 0.0.6) ruby depends on
        faraday (~> 0.8.4) ruby

    engagement (>= 0) ruby depends on
      twitter (~> 5.9.0) ruby depends on
        faraday (0.9.0)
```

**Comment by cdmwebs** [comment]
I've fixed this in the most recent upgrade, but Fakeweb will be removed from VCR. I'll take a look at fixing that one.

**Comment by cdmwebs** [comment]
Closed in 5e4913c5bbdd870932.
